lib/routes/security_pin/lock_screen.dart

```diff
@@ -30,9 +31,7 @@ class _AppLockScreenState extends State<AppLockScreen> {
   @override
   Widget build(BuildContext context) {
     return WillPopScope(
-      onWillPop: () {
-        return Future.value(widget.canCancel);
-      },
+      onWillPop: willPopCallback(context),
```

Comment:
@ademar111190 we should return true if widget.canCancel is true as there is another flow that uses the lock screen and let the user cancel (go back).

Reply:
@roeierez I did add the `can cancel` clause to the `willPopCallback` function, I guess now it reflects correctly the behaviour.